apps/condo/domains/billing/schema/BillingReceiptFile.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Generated by `createschema billing.BillingReceiptFile 'file:File;context:Relationship:BillingIntegrationOrganizationContext:CASCADE;receipt:Relationship:BillingReceipt:CASCADE;controlSum:Text'`
+ */
+
+const { Text, Relationship, File } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/billing/access/BillingReceiptFile')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const BILLING_RECEIPT_FILE_FOLDER_NAME = 'billingReceipt'
```

Comment:
s3 will lowercase folder name
better to use 'billing-receipt' or 'billing-receipt-pdf'